docs/user-guide: document the new installer.modules customization (#100)
osbuild/osbuild.github.io: docs/user-guide/01-blueprint-reference.md

Note that osbuild-composer will automatically add the command that installs the system (`liveimg` or `ostreesetup`) if one is relevant for the image type (`image-installer` and `edge-installer`/`iot-installer` respectively), so this line or any line that conflicts with it should not be included. See the relevant [Kickstart documentation](https://pykickstart.readthedocs.io/en/latest/kickstart-docs.html#ostreecontainer) for more information.
This customization cannot be used in combination with any other installer customizations.

#### Anaconda Modules 🔵 {#installer-kickstart}

The Anaconda installer can be configured by enabling or disabling its dbus modules.

<Tabs values={tabValues} >
<TabItem value="on-premises" >
```toml
[customizations.installer.modules]
enable = []
disable = []

> """

Reviewer: is this a typo?

Author: It was.  Thanks!

Note that osbuild-composer will automatically add the command that installs the system (`liveimg` or `ostreesetup`) if one is relevant for the image type (`image-installer` and `edge-installer`/`iot-installer` respectively), so this line or any line that conflicts with it should not be included. See the relevant [Kickstart documentation](https://pykickstart.readthedocs.io/en/latest/kickstart-docs.html#ostreecontainer) for more information.
This customization cannot be used in combination with any other installer customizations.

> #### Anaconda Modules 🟣 {#installer-kickstart}

Reviewer 2: The anchor needs to be unique e.g.
```suggestion
#### Anaconda Modules 🟣 {#installer-kickstart-anaconda-modules}
```

> The Anaconda installer can be configured by enabling or disabling its dbus modules.

Reviewer 2: nitpick - is it DBus or D-Bus?
```suggestion
The Anaconda installer can be configured by enabling or disabling its D-Bus modules.
```

> <Tabs values={tabValues} >

Reviewer 2: Please change to `tabValuesBootcOnly` and make `bootc` the default
```suggestion
<Tabs values={tabValuesBootcOnly} defaultValue="bootc">
```

where you need to implement this in `src/components/Highlight.js` first and import here 😅
```
export const tabValuesBootcOnly = [
    { label: '⚪ on premises', value: 'on-premises', attributes: { className: 'highlight tab disabled' } },
    { label: '⚪ hosted', value: 'hosted', attributes: { className: 'highlight tab disabled' } },
    { label: '🟣 bootc', value: 'bootc', attributes: { className: 'highlight tab bootc' } },
];
```

Author: Very nice, thanks.  Added!

<TabItem value="on-premises" >
```
ℹ️ - Currently not supported
```
</TabItem>
<TabItem value="hosted" >
```
ℹ️ - Currently not supported
```
</TabItem>
<TabItem value="bootc" >
```json
{
  "customizations": {
    "installer": {
      "modules": {
        "enable": [
          "org.fedoraproject.Anaconda.Modules.Localization"
        ],
        "disable": [
          "org.fedoraproject.Anaconda.Modules.Users"
        ]
      }
    }
  }
}
```
</TabItem>
</Tabs>

The following module names are known and supported:
- `org.fedoraproject.Anaconda.Modules.Localization`
- `org.fedoraproject.Anaconda.Modules.Network` 
- `org.fedoraproject.Anaconda.Modules.Payloads`
- `org.fedoraproject.Anaconda.Modules.Runtime`
- `org.fedoraproject.Anaconda.Modules.Security`
- `org.fedoraproject.Anaconda.Modules.Services`
- `org.fedoraproject.Anaconda.Modules.Storage`
- `org.fedoraproject.Anaconda.Modules.Subscription`
- `org.fedoraproject.Anaconda.Modules.Timezone`
- `org.fedoraproject.Anaconda.Modules.Users`

> *Note: The values are not validated. Any name listed under `enable` will be added to the Anaconda configuration. This way, new or unknown modules can be enabled. However, it also means that mistyped or incorrect values may cause Anaconda to fail to start.*

Reviewer 2: maybe emphasize this more?
```suggestion
*Note: The values are **not validated**. Any name listed under `enable` will be added to the Anaconda configuration. This way, new or unknown modules can be enabled. However, it also means that mistyped or incorrect values may cause Anaconda to fail to start.*
```

Author: Good idea.
